src/test/java/com/xfers/model/example/application/SampleLoan.java

```diff
@@ -28,14 +28,21 @@
     private static String refno = ""; // for creating new loan
     private static String disbursementIdempotencyId = ""; // for creating new disbursement
     private static String repaymentIdempotencyId = ""; // for creating new loan repayment
-
+    
+    /**
+    * This sample code show cases the step required to request a loan > 
```

Comment:
This comment is incomplete, below is the usage of this main function. Also, the example below's grammar needs to be corrected.

```
the steps required to create an Xfers User, then upload KYC data,
request a loan > if approved create a disbursement to the end user.
Last, also repayment flow
```